### ISSUE TITLE
Enhancement offer: CIPRES analyses?

### ISSUE THREAD

**Issue by rvosa** [hCoV-2019/pangolin#12]
Hi,

I've been working on some [tooling](https://github.com/naturalis/covid19-phylogeny) to have the sequence alignment (MAFFT) and tree inference (IQTree) steps delegated to the [CIPRES REST](https://www.phylo.org/restusers/documentation) portal. The general idea is that especially the alignment step is a bit expensive for people to run locally so they might want to offload that to the cloud, and I understand from the CIPRES PI that they give preferential treatment to SARS-Cov-2 analyses right now.

With a bit of effort, I should be able to make it so that the tool is a bioconda package that you could use in place of the local steps you have [here](https://github.com/hCoV-2019/pangolin/blob/master/pangolin/scripts/prepare_package_data.smk#L44) and [here](https://github.com/hCoV-2019/pangolin/blob/master/pangolin/scripts/prepare_package_data.smk#L95), but the upshot is that users would have to get a user account and app key registration on the CIPRES server.

In other words: better performance but with somewhat more complexity. Is this something you care for?

**Comment by aineniamh** [comment]
It's an interesting idea. We have [Anthony Underwood](https://github.com/aunderwo) developing a web app that would simplify from the users perspective. In terms of compute required for pangolin to run locally, we're not running the MSA or tree from scratch, just running a profile alignment and then iqtree with the guide tree. Really appreciate the offer though, but I think we're going in the direction of a web-app for those who don't have the compute required!
